doc/source/reference/c-api.coremath.rst

```diff
@@ -81,7 +81,22 @@ Useful math constants
 ~~~~~~~~~~~~~~~~~~~~~
 
 The following math constants are available in npy_math.h. Single and extended
-precision are also available by adding the F and L suffixes respectively.
+precision are also available by adding the f and l suffixes respectively, with the exceptions of the following: -
+-> NPY_INFINITYF
+-> NPY_NANF
+-> NPY_PZEROF
+-> NPY_NZEROF
+-> NPY_INFINITYL
+-> NPY_NANL 
+-> NPY_PZEROL
+-> NPY_NZEROL
+
+/*
+ * Useful constants
+ */
+#define NPY_E         2.718281828459045235360287471352662498  /* e */
+#define NPY_LOG2E     1.442695040888963407359924681001892137  /* log_2 e */
```

Comment:
Note that the documentation for `NPY_LOG2E` and `NPY_LOG10E` below is a bit confusing, as I expect far more people to be familiar with `e` than its formal name of Euler-Mascheroni constant. We should slip `e` in there somehow, maybe just use `e` and put Euler-Mascheroni in parenthesis.